app/src/main/java/com/android/sample/ui/activity/ActivityForm.kt

```diff
@@ -0,0 +1,400 @@
+import com.android.sample.resources.C.Tag.SMALL_PADDING
+import com.android.sample.resources.C.Tag.STANDARD_PADDING
+import com.android.sample.resources.C.Tag.TOP_TITLE_SIZE
+import com.android.sample.ui.camera.Carousel
+import com.android.sample.ui.components.AttendantPreview
+import com.android.sample.ui.components.MyDatePicker
+import com.android.sample.ui.components.MyTimePicker
+import com.android.sample.ui.dialogs.AddUserDialog
+import com.google.firebase.Timestamp
+import com.vanpra.composematerialdialogs.MaterialDialogState
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ActivityForm(
+    context: Context,
+    selectedImages: List<Bitmap>,
+    onOpenDialogImage: () -> Unit,
+    onDeleteImage: (Bitmap) -> Unit,
+    onTitleChange: (String) -> Unit,
+    title: String,
+    maxTitleSize: Int,
+    onDescriptionChange: (String) -> Unit,
+    description: String,
+    maxDescriptionSize: Int,
+    onClickDate: () -> Unit,
+    onCloseDate: (MaterialDialogState) -> Unit,
+    onSelectDate: (Timestamp) -> Unit,
+    dueDate: Timestamp,
+    dateIsOpen: Boolean,
+    dateIsSet: Boolean,
+    onClickStartingTime: () -> Unit,
+    startTimeIsOpen: Boolean,
+    startTimeIsSet: Boolean,
+    onStartTimeSelected: (Timestamp) -> Unit,
+    startTime: String,
+    onCloseStartTime: (MaterialDialogState) -> Unit,
+    onClickDurationTime: () -> Unit,
+    durationIsOpen: Boolean,
+    durationIsSet: Boolean,
+    onSelectDuration: (Timestamp) -> Unit,
+    onCloseDuration: (MaterialDialogState) -> Unit,
+    duration: String,
+    price: String,
+    onPriceChange: (String) -> Unit,
+    placesMax: String,
+    onPlacesMaxChange: (String) -> Unit,
+    locationQuery: String,
+    onLocationQueryChange: (String) -> Unit,
+    showDropdown: Boolean,
+    locationSuggestions: List<Location?>,
+    onDismissLocation: () -> Unit,
+    onLocationClick: (Location) -> Unit,
+    expandedType: Boolean,
+    onExpandedTypeChange: (Boolean) -> Unit,
+    onSelectType: (ActivityType) -> Unit,
+    onDismissType: () -> Unit,
+    selectedOptionType: String,
+    expandedCategory: Boolean,
+    onExpandedCategoryChange: (Boolean) -> Unit,
+    onDismissCategory: () -> Unit,
+    selectedOptionCategory: Category?,
+    selectedOptionInterest: String?,
+    expandedInterest: Boolean,
+    onInterestExpandChange: (Boolean) -> Unit,
+    onInterestDismiss: () -> Unit,
+    onInterestSelect: (String) -> Unit,
+    onSelectCategory: (Category) -> Unit,
+    attendees: List<User>,
+    showDialogUser: Boolean,
+    deleteAttendant: (User) -> Unit,
+    onDismissUserDialog: () -> Unit,
+    onAddUser: (User) -> Unit,
+    onOpenUserDialog: () -> Unit,
+    onProfileClick: (User) -> Unit,
+    imageViewModel: ImageViewModel,
+) {
+  Carousel(openDialog = onOpenDialogImage, itemsList = selectedImages, deleteImage = onDeleteImage)
+  Spacer(modifier = Modifier.height(STANDARD_PADDING.dp))
+  RemainingPlace(title, maxTitleSize)
+  OutlinedTextField(
```

Comment:
Please use the already created 'OutlinedTextFieldWithErrorState' provided in TextFIelds.kt , or if you want to avoid the error states (even if I think they are important here), you can duplicate OutlinedTextFieldWithErrorState and create the same text field but without error state

Reply:
Was going to refactor this part in next PRs when I will be improving UI, for the moment I wanted to totally change the precedent format of both classes